packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -38,6 +38,9 @@ export interface AccordionTitleProps
   /** If at least one panel needs to stay active and this title does not correspond to the last active one. */
   canBeCollapsed?: boolean
 
+  /** Disabled - does not react on click events */
```

Comment:
/** An accordion title can show it is currently unable to be interacted with. */